UPGRADE.md

```diff
@@ -1,5 +1,12 @@
 # Upgrade to 3.0
 
+## BC BREAK: Changes in the `Doctrine\DBAL\Schema` API
+
+- Method `Doctrine\DBAL\Schema\AbstractSchemaManager::_getPortableViewDefinition()` now optionally returns null instead of false.
```

Comment:
Is there a valid case where it would return `false`? In my version, the default behavior was `throw DBALException::notSupported('Views');`.

Reply:
I will see if I can work that out.

Reply:
Good call. Fixed here https://github.com/doctrine/dbal/pull/3565/commits/02d4639428a6714ad8fa2a57869916440cffa19b